var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -588,6 +590,12 @@ def cmake_args(self):
             '-DPYTHON_EXECUTABLE:PATH={0}'.format(spec['python'].command.path),
         ]
 
+        if '+cuda' not in spec:
+            cmake_args.extend([
+                '-DCUDA_TOOLKIT_ROOT_DIR:PATH=IGNORE',
+                '-DCUDA_SDK_ROOT_DIR:PATH=IGNORE',
+                '-DCUDA_NVCC_EXECUTABLE:FILEPATH=IGNORE'])
+
```

Comment:
You'll also want to add `-DLIBOMPTARGET_DEP_CUDA_DRIVER_LIBRARIES=IGNORE`

Reply:
Thanks!  Added.